tests/integration/staking/keeper/genesis_test.go

```diff
@@ -196,7 +196,7 @@ func TestInitGenesisLargeValidatorSet(t *testing.T) {
 	bondedPoolAmt := math.ZeroInt()
 	for i := range validators {
 		validators[i], err = types.NewValidator(
-			sdk.ValAddress(addrs[i]),
+			sdk.ValAddress(addrs[i]).String(),
```

Comment:
although this is fine, feels dirty / bad practice

Reply:
oh this is 100% ugly, mainly didnt want to rewrite all these tests to use the codec as we will be rewriting this module in the near future